actioncable/lib/action_cable/connection/base.rb

```diff
@@ -2,7 +2,7 @@
 
 module ActionCable
   module Connection
-    # For every WebSocket the Action Cable server accepts, a Connection object will be instantiated. This instance becomes the parent
+    # For every WebSocket connection the Action Cable server accepts, a Connection object will be instantiated. This instance becomes the parent
```

Comment:
👍 for this change, its better to be explicit. See https://www.wikiwand.com/en/WebSocket for usage.